test/test_integration_credentials_keyring.py

```diff
@@ -378,6 +379,19 @@ def test_keyring_locked_after_login_will_raise_error_keyring(self):
         with patch('conjur.wrapper.keystore_adapter.KeystoreAdapter.is_keyring_accessible', return_value=False):
             self.invoke_cli(self.cli_auth_params, ['list'], exit_code=1)
 
+    '''
+    Validate env variable is set correctly
+    '''
+    @integration_test(True)
+    def test_keyring_locked_after_login_will_raise_error_keyring(self):
+        _ = utils.create_cred_store()
+        if platform.system() == "Darwin":
```

Comment:
Move all this hardcoded strings to consts

Reply:
100%
anyway I added a util function that will find the current os and return an os enum type